arches/install/arches-templates/project_name/urls.py-tpl

```diff
@@ -7,6 +10,11 @@ urlpatterns = [
     path('', include('arches.urls')),
 ] + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
 
-# if settings.SHOW_LANGUAGE_SWITCH is True:
-#     urlpatterns = i18n_patterns(*urlpatterns)
-#     urlpatterns.append(path("i18n/", include("django.conf.urls.i18n")))
\ No newline at end of file
+if settings.SHOW_LANGUAGE_SWITCH is True:
+    # Only handle i18n routing in active project. This will still handle the routes provided by Arches core and Arches applications,
+    # but handling i18n routes in multiple places causes application errors.
+    directory_path = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))

```

Comment:
Thanks for jumping on this. I feel like we run into subtle issues with editable installs and virtualization and symlinks often enough that I'm wondering if we can skin this apple another way, maybe like:
```py
if settings.APP_NAME != "Arches" and settings.APP_NAME not in settings.ARCHES_APPLICATIONS
```
Would that be equivalent?